pkg/config/config.go

```diff
@@ -376,12 +326,15 @@ var DefaultHooksDirs = []string{"/usr/share/containers/oci/hooks.d"}
 // might change in the future.
 func NewConfig(userConfigPath string) (*Config, error) {
 
-	config := &Config{} // start with an empty config
+	config, err := DefaultConfig()
+	if err != nil {
+		return nil, err
+	}
```

Comment:
Can you add a rather verbose comment about the toml semantics and how the overwriting works and that starting with the hard-coded default config makes sure we're always having something sane?

Reply:
Fixed